fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/scan/cli/cmd/FoDScanStartMobileCommand.java

```diff
@@ -55,7 +50,8 @@ public class FoDScanStartMobileCommand extends AbstractFoDJsonNodeOutputCommand
     @Mixin private FoDReleaseByQualifiedNameOrIdResolverMixin.PositionalParameter releaseResolver;
     private enum MobileAssessmentTypes { Mobile, MobilePlus, Remediation }
     @Option(names = {"--assessment-type"}, required = true)
-    private MobileAssessmentTypes mobileAssessmentType;
+    //private MobileAssessmentTypes mobileAssessmentType;
+    private String mobileAssessmentType;
```

Comment:
Same comment as for DAST, although we can't implement this for now due to FoD lacking `GET scan-setup` for mobile. As discussed, this command should be hidden/marked as PREVIEW in usage header (and with explanation in usage description)

Reply:
As above